AmpegManualTestProject.txt

```diff
@@ -0,0 +1,104 @@
+--Ampeg Manual Test Project--
+https://ampeg.com/
+
+1. You decide what should be priority (to test) in the webapp.
+2. 6 manual test cases total.
+3. Ensure to include what 'should' be included in a manual test (title, summary, etc).
+4. Ensure that 5 tests consist of navigating to more than one page.
+5. Ensure that 2 tests consist of navigating to 3 or more pages.
+
+1:
+
+Title: Find Contact Info
+
+Summary: The user will navigate the website and discover the company's contact info.
+
+Pre-conditions: Start at the home page.
+
+Traceability: User is able to get contact info if needed.
+
+Steps: Be at the home page and click on "Support" at the top right. Navigate to the bottom and click on "Contact Us".
+
+Expected result: User should be on a page listed with the company's contact info.
+
+2:
+
+Title: Create account
+
+Summary: The user will make an account if they dont have one.

```

Comment:
I would reword this to describe what is being tested.
Don't use conditionals in your tests (don't use IF statements)
Your test could be two tests actually.

Summary: Verifies that a new account is created when none exists for the user.